apis/vshn/v1/dbaas_vshn_postgresql.go

```diff
@@ -108,14 +109,15 @@ type VSHNDBaaSNetworkSpec struct {
 	IPFilter []string `json:"ipFilter,omitempty"`
 }
 
-// VSHNDBaaSBackupSpec contains settings to control the backups of an instance.
-type VSHNDBaaSBackupSpec struct {
-	// +kubebuilder:validation:Pattern="^([0-1]?[0-9]|2[0-3]):([0-5][0-9]):([0-5][0-9])$"
-	// +kubebuilder:default="21:30:00"
+type VSHNPostgreSQLBackup struct {
+	// +kubebuilder:validation:Pattern=^(\*|([0-9]|1[0-9]|2[0-9]|3[0-9]|4[0-9]|5[0-9])|\*\/([0-9]|1[0-9]|2[0-9]|3[0-9]|4[0-9]|5[0-9])) (\*|([0-9]|1[0-9]|2[0-3])|\*\/([0-9]|1[0-9]|2[0-3])) (\*|([1-9]|1[0-9]|2[0-9]|3[0-1])|\*\/([1-9]|1[0-9]|2[0-9]|3[0-1])) (\*|([1-9]|1[0-2])|\*\/([1-9]|1[0-2])) (\*|([0-6])|\*\/([0-6]))$
```

Comment:
@glrf I'm not sure if Exoscale actually supports those for their schedules anyway. We'd need to check that.

Reply:
so remove validation? I'm not sure if I can validate all of these using regex

Reply:
What matters is what stackgres supports, right?  And at least lists and ranges are very standard. An all of these features come up on the crontab man page..

Reply:
https://stackgres.io/doc/latest/reference/crd/sgbackupconfig/

scroll down to cronSchedule field

Reply:
O wow, what broken cron library are they using...  Did you check that they don't support lists and ranges, because I'd expect these to work usually...

But I guess if they really don't support it, this should be fine

Reply:
> What matters is what stackgres supports, right?

Yeah sorry, meant StackGres but it wrote Exoscale 🙈